### DISCUSSION TITLE
MacOs and Linux support status

### DISCUSSION THREAD

**Discussion by MindWrapper** [stryker-mutator/stryker-net#1415] [Q&A] [answered]
Just wanted to check if MacOs and Linux are supported or if there are any plans to support them?
If no, I can try to implement it.

**Comment by rouke-broersma** [comment, marked as answer]
Welcome! Linux and macos are supported for dotnet core but not for framework (for obvious reasons). We integration test all releases on windows, macos and Linux. Though our development is done on windows as we don't have mac/Linux workstations. If you find any issues with macos and/or linux specific workflows feel free to report them as bugs and we will try to solve the problems. Of course if you can provide the fixes yourself this is also welcomed.
